desktop/apps/editorial_features/components/gucci/templates/head.jade

```diff
@@ -0,0 +1,33 @@
+if sd.SUB_PATH
+  - page = curation.get('sections')[sd.VIDEO_INDEX]
+else
+  - page = sd.CURATION
+
+- url = sd.APP_URL + '/gender-equality' + sd.SUB_PATH
+
```

Comment:
Just added a slug for each section, so you can ask for `page.slug` here rather than setting SUB_PATH

Reply:
👌 Cool, updated it with a `IS_NESTED_PATH` since we still need to distinguish between first video and landing page metadata